ciao-controller/openstack_compute.go

```diff
@@ -359,27 +361,29 @@ func (c *controller) CreateServer(tenant string, server compute.CreateServerRequ
 		nInstances = server.Server.MinInstances
 	}
 
+	if server.Server.Name != "" {
+		r := regexp.MustCompile("^[a-z0-9-]*$")
```

Comment:
do you think we should put a limit on the length for the name as well? wondering if someone could be evil here.